src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -356,3 +357,20 @@ def test_pydivert(pyi_builder):
         import pydivert
         pydivert.WinDivert.check_filter("inbound")
         """)
+
+
+@importorskip('skimage')
+@pytest.mark.skipif(not is_module_satisfies('skimage >= 0.16.0'),
+                    reason='The test supports only skimage 0.16.0 or newer.')
+def test_skimage(pyi_builder):
+    # Explicitly import all skimage sub-packages.
+    # The list below is valid for skimage >= 0.16.0
+    subpackages = ['color', 'data', 'draw', 'exposure', 'feature',
+                   'filters', 'future', 'graph', 'io', 'measure',
+                   'metrics', 'morphology', 'registration',
+                   'restoration', 'segmentation', 'transform',
+                   'util', 'viewer']
+
+    code = "\n".join(["import skimage.{0}".format(x) for x in subpackages])
```

Comment:
I'd rather run each of these as separate tests per submodule. That way we'd know of a hidden dependency of one is being masked because another submodule which also requires the hidden dependency. but either does it explicitly or has a hook.

Reply:
Probably would take a while to run though...

Reply:
I initially had separate tests for each submodule, but all 36 of them took a while to run, so I figured a combined test (effectively 2 tests) might be a better idea. But I agree that separate tests would be better indicators for issues, so I can split them up again, if we don't mind the prolonged test time.

Reply:
Hmm, well I certainly don't want all 36 of them running as part of our main CI. Personally, I'd rather we split them but either omit `scikit-image` from our requirements file or create a `@too_slow_for_CI` marker to skip them for our regular per-push CI. And then just test the slow test once and forget about it. @Legorooj What do you think?

Reply:
Remove it from the requirements, and use the `pytest.parameterize` decorator to parameterize the test. We can maybe schedule a build or something that runs _every_ test but we can look at that later.

Reply:
OK, I put the submodules into `pytest.parametrize` again, so we have a separate test for each one. And I couldn't find `scikit-image` anywhere in requirements, so it shouldn't be slowing the tests by default.